languages/java/exercises/concept/booleans/src/test/java/QuestLogicTest.java

```diff
@@ -0,0 +1,316 @@
+import org.junit.Test;
+import org.junit.Ignore;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class QuestLogicTest
+{
+    @Test
+    public void cannot_execute_fast_attack_if_knight_is_awake()
+    {
+        var knightIsAwake = true;
```

Comment:
`var` has been introduced with Java 10. At the moment we are introducing only the _classical_ concept (we haven't introduced all _new_ concepts present in Java 8 and later)
use explicit type for variables

Reply:
fixed